toucan_connectors/toucan_connector.py

```diff
@@ -32,6 +32,7 @@ class DataStats(NamedTuple):
     execution_time: Optional[float] = None  # query's execution time in ms
     conversion_time: Optional[float] = None  # Result conversion to DataFrame time
     df_memory_size: Optional[int] = None  # Dataframe's memory usage in bytes
+    metadata: Dict[str, Any] = {}
```

Comment:
Maybe we should just name that `others`? So it's accessible using slice.stats.others
I'm starting to fear that we use the term `meta` in a lot of places with different meanings. As it doesn't mean something in its own, it could be confusing

Reply:
(or something else if you have a better idea)

Reply:
this is a good remark, `meta` is used in a lot of place in the codebase.

other is fine